pages/api/events.js

```diff
@@ -53,5 +67,10 @@ export default async function handler(req, res) {
     .filter((event) => new Date(event.date.end) > new Date())
     .sort((a, b) => new Date(a.date.start) - new Date(b.date.start));
 
-  res.status(200).json(eventsFiltered);
+  return {
+    statusCode: 200,
+    events: {
+      eventsArray: eventsFiltered
+    }
```

Comment:
Having this nested with the type I don't think is consistent with similar parts of the code?

Reply:
The problem is this is the only one I have updated so far that didn't return an object.

In order to have a place to put an error message, I return an object.
The object contains the array on success or the error message on failure

Reply:
ah ok, what about having the error message outside the data, similar to `status`?

Reply:
Doing it that way would make it more difficult to work with the existing API. 

It make more sense for me for an API to return a status code & JSON payload.
This puts the responsibility on the caller to check the status code & evaluate the JSON payload depending on that status.